terragrunt/aws/vpc/vpc.tf

```diff
@@ -0,0 +1,38 @@
+module "vpc" {
+  source            = "github.com/cds-snc/terraform-modules?ref=v0.0.46//vpc"
```

Comment:
Could you change the `ref` tag to the latest version in the `terraform-modules` repo.